DATA/EQUIPMENT/engine_equip.ini

```diff
@@ -3258,9 +3258,9 @@ ids_info = 460270
 ; Developed by the Government supported avionics supplier \iRoyal Avionics\I, this engine serves as a reliable asset to all heavy Bretonian armed forces.\n
 volume = 0
 mass = 20
-max_force = 8000000
+max_force = 24000000
 linear_drag = 1
-reverse_fraction = 0.5
+reverse_fraction = 5.5

```

Comment:
Why should the ship go 550% of its forward speed in reverse?

Reply:
Confirmed and changed.